_topic_map.yml

```diff
@@ -119,6 +119,23 @@ Topics:
   - Name: Autoscaling worker nodes
     File: rosa-autoscale-nodes
 ---
+Name: Monitoring user workloads
+Dir: monitoring
+Distros: openshift-rosa
+Topics:
+- Name: Understanding the monitoring stack
+  File: osd-understanding-the-monitoring-stack
+- Name: Configuring the monitoring stack
```

Comment:
we could keep it since the OSD admin could configure the user workload monitoring

Reply:
Ok, keeping it.

Reply:
user-workload-monitoring is enabled by default, seems that OSD/ROSA user needn't change cm cluster-monitoring-config, otherwise this part should be kept. Even the dedicated-admin user only has default permissions to configure and access monitoring for user-defined projects, which means create namespace, deploy service and create servicemonitor.